packages/application-components/src/components/page-content-containers/page-content-narrow/page-content-narrow.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactNode } from 'react';
+import styled from '@emotion/styled';
+
+export type TPageContentNarrow = {
+  children: ReactNode;
+};
+
+const LeftContentMarginColumn = styled.div`
+  grid-area: 'left-margin';
+`;
+
+const MainContentColumn = styled.div`
+  grid-area: 'main-content';
+`;
+
+const RightContentMarginColumn = styled.div`
+  grid-area: 'right-margin';
+`;
+
+const MainContainer = styled.section`
```

Comment:
Shouldn't be a `<section>` only the container that renders the content? 🤔

Reply:
Yes, you're totally right.

Reply:
Updated in this commit: 5e306e1c1fabf0f61996604f1fb07d166f41e12e